app/nginx/sites-enabled/api.couchers.org

```diff
@@ -0,0 +1,30 @@
+server {
+    listen 443 http2 ssl default_server;
+
+    server_name api.couchers.org;
+
+    access_log /var/log/nginx/api.couchers.org.access.log;
+    error_log /var/log/nginx/api.couchers.org.error.log;
+
+    ssl_certificate /certs/live/api.couchers.org/fullchain.pem;
+    ssl_certificate_key /certs/live/api.couchers.org/privkey.pem;
+
+    location /org.couchers. {
+        proxy_http_version 1.1;
+        proxy_pass http://envoy:8888;
+    }
+
+    location = / {
+        add_header Content-Type text/plain;
+
+        set $msg "This is the API for the Couchers.org app. If you're interested in helping out in building the";
+        set $msg "${msg} next-generation couch-surfing platform with us, or if you're just interested in API access,";
+        set $msg "${msg} please email us.\n";
```

Comment:
text/plain usually opens up a download dialog box for me, maybe text/html. And direct them to the repo with the API description.

Reply:
Would you mind checking, what browser are you on? https://api-test.coucher.org/

`text/plain` should be fine?

Reply:
Fixefox, that link works fine thuogh, it is displayed in the browser window without dialog. It sends double Content-Type though?
```
HTTP/1.1 200 OK
Server: nginx
Date: Fri, 20 Nov 2020 22:59:08 GMT
Content-Type: application/octet-stream
Content-Length: 203
Connection: keep-alive
Content-Type: text/plain
```

Reply:
Yes, I don't know why it's doing that. I don't think this is a big deal for now. I can change it to `text/html` if you wish? (But it's not HTML, it's plaintext).

Reply:
Stay with `text/plain` as it worked for me :)

Reply:
(I kinda like the teletype font, looks old school ahah ;) )